backend/drm/drm.c

```diff
@@ -1255,6 +1255,15 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 			}
 		}
 
+		memset(&wlr_conn->props, 0, sizeof(wlr_conn->props));
+		get_drm_connector_props(drm->fd, wlr_conn->id, &wlr_conn->props);
+
+		size_t edid_len = 0;
+		uint8_t *edid = get_drm_prop_blob(drm->fd,
+			wlr_conn->id, wlr_conn->props.edid, &edid_len);
+		parse_edid(&wlr_conn->output, edid_len, edid);
```

Comment:
We need to signal the fact that the EDID changed, if it changed. I think the easiest way to do it is by removing and re-adding the connector.

Reply:
Wouldn't this also mean that the optimization to not modeset existing outputs is lost? If I remove and re-add the connector, a new crtc will get allocated and the compositor has to restart rendering on the new output.

Reply:
When the EDID changes, it means the monitor has been swapped, ie. supported resolutions changed. Maybe we can skip modesets if `memcmp(cur_mode, new_mode) == 0`?

Reply:
Note: we only need to signal if something has changed.